### DISCUSSION TITLE
Table x-filament blade components

### DISCUSSION THREAD

**Discussion by clinvest** [filamentphp/filament#11203] [Help] [answered]
### Package

Other

### Package Version

v3.0.0

### How can we help you?

Hi all,

Is there a way to use the filament table components? something like `<x-filament::table>` etc. for rows, td, tr...

I don't find anything about it in the documentation, neither in the source code.

Thank you in advance,

**Comment by leandrocfe** [comment, marked as answer]
https://github.com/filamentphp/filament/issues/7105

**Reply by IgorDePaula** [reply]
No docs on official site.
